OneSignalExample/Assets/OneSignal/CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - iOS crash when calling OneSignal.User.PushSubscription.Id and OneSignal.User.PushSubscription.Token when they are null.
 
+## [5.1.0]
+### Fixed
+- Add getters for the user state that includes onesignalId and externalId
+- Add observer for the user state
```

Comment:
I think we could also specify the name of the event handler. Something like, "Add public event handler OneSignal.User.Changed that fires when the OnesignalId or ExternalId changes"

I also think this should be categorized under ###Changed